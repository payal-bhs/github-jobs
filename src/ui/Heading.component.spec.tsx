import React from "react";
import { shallow } from "enzyme";
import { Heading } from "./Heading.component";


describe("Heading", () => {
    it("render project header", () => {
        const header = shallow(<Heading />);
        expect(header).toMatchSnapshot();
    });
});