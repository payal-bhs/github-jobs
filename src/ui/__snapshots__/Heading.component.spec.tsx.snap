// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading render project header 1`] = `ShallowWrapper {}`;
